libs/styles/styles.css

```diff
@@ -553,7 +553,17 @@ span.icon.margin-right { margin-right: 8px; }
 
 span.icon.margin-left { margin-left: 8px; }
 
-span.icon.margin-inline-end { margin-inline-end: 8px; }
+span.icon.margin-inline-end { 
+  -moz-margin-end: 8px;
+    -webkit-margin-end:8px;
```

Comment:
Are the prefixes actually needed? Based on caniuse, the browser versions requiring them go back to 2018, we'd very likely have much bigger issues on those browsers than alignment :)

<img width="627" alt="Screenshot 2024-05-21 at 17 53 40" src="https://github.com/adobecom/milo/assets/11267498/80ddea37-be2b-429d-a0a8-119087e203f0">

Reply:
@overmyheadandbody 
Make sense.
Prefix are removed.